### PR TITLE
Pin nokogiri version

### PR DESCRIPTION
**Pull request by rajiv-g**

### CODE REVIEW

File: sensu-plugins-passenger.gemspec [hunk 1]
```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |s| # rubocop:disable Metrics/BlockLength
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsPassenger::Version::VER_STRING
 
-  s.add_runtime_dependency 'nokogiri',     '~> 1.8', '>= 1.8.2'
+  s.add_runtime_dependency 'nokogiri',     '~> 1.10.0'
```
Comment (mike-stewart) [inline sensu-plugins-passenger.gemspec]: Can you pin to 1.11? There’s a vulnerability in 1.10

**Review by mike-stewart — APPROVED** [review]

File: sensu-plugins-passenger.gemspec (hunk 1 above)
Comment (rajiv-g) [reply sensu-plugins-passenger.gemspec]: But they dropped support for 2.3 & 2.4 ruby version

Comment (rajiv-g) [reply sensu-plugins-passenger.gemspec]: Shall we use 1.8.X?

Comment (mike-stewart) [reply sensu-plugins-passenger.gemspec]: Use the latest version that will work then

Comment (rajiv-g) [reply sensu-plugins-passenger.gemspec]: Update PR to use version 1.8.5

**Pull request merged**